modules/ROOT/pages/_partials/commons/common-query-n1ql-mobile.adoc

```diff
@@ -1638,27 +1638,144 @@ Returns `NULL` if either or both expressions are `NULL`.+
 === Date and Time Functions
 
 .Date and Time Functions
-[tbl-func-agg,cols="1m,4", options="header"]
+[cols="5*", options="header"]
 |===
 
-|Function
-|Description
+|Function |Description |Arguments |Return Value |Further Information
 
-|STR_TO_MILLIS(expr)
-|Returns the number of milliseconds since the unix epoch of the given ISO 8601 date input string.
+|`STR_TO_MILLIS(date1)`
+|Coverts a date string to Epoch/UNIX milliseconds.
+a|
 
-|STR_TO_UTC(expr)
-|Returns the ISO 8601 UTC date time string of the given ISO 8601 date input string.
+* `date1` - A valid date string.
+|Returns an integer containing the converted date string into Epoch/UNIX milliseconds.
+|
+|`STR_TO_UTC(date1)`
+|Converts a date string into the equivalent date in UTC.
+a|
 
-|MILLIS_TO_STR(expr)
-|Returns a ISO 8601 date time string in device local timezone of the given number of milliseconds since the unix epoch expression.
+* `date1` - A valid date string
+|Returns a date string representing the date string converted to UTC.
+|The output date format follows the date format of the input date. 
+If `date1` is not a valid date format, `null` is returned.
+|`STR_TO_TZ(date1, tz)`
+|Converts a date string to it's equivalent in the specified timezone.
+a|
 
-|MILLIS_TO_UTC(expr)
-|Returns the UTC ISO 8601 date time string of the given number of milliseconds since the unix epoch expression.
+* `date1` - A valid date string. 
+This is converted to UTC.
 
-|===
+* `tz` - An integer that represents minutes offset from UTC.
+For example, `UTC-5` would be represented as `-300`.
+|Returns a date string representing the date string converted to the specified timezone.
+|If `date1` is not a valid date format, `null` is returned.
+|`MILLIS_TO_STR(date1)`
+|Converts an Epoch/UNIX timestamp into the specified date string format.
+a|
+
```

Comment:
this newline is required because of asciidoc weirdness?

Reply:
Yeah, it won't recognise the formatting otherwise